yarn.lock

```diff
@@ -7852,6 +7852,15 @@ enhanced-resolve@^4.0.0, enhanced-resolve@^4.1.0:
     memory-fs "^0.4.0"
     tapable "^1.0.0"
 
+enhanced-resolve@^4.1.1:
```

Comment:
__optional nit:__ it seems you can merge this new `enhanced-resolve@^4.1.1` declaration with the existing `enhanced-resolve@^4.1.0, enhanced-resolve@^4.1.1`?
```
enhanced-resolve@^4.0.0, enhanced-resolve@^4.1.0, enhanced-resolve@^4.1.1:
  version "4.5.0"
```

Reply:
Oops I missed that, I'll do it now thx